### ISSUE TITLE
Event "visit the page" in duplicate

### ISSUE THREAD

**Issue by claracousin1** [mautic/mautic#12148]
### Mautic Version

4.4.x series

### PHP version

7.4

### What browsers are you seeing the problem on?

Not relevant

### What happened?

Hello everybody,

This concerns the history of a contact. When a contact visits a page on a website from a sent e-mail, the event's "page visit"  appears twice with a few seconds interval.

The first event is the URL of a click from an e-mail. 
The second event is the title of the page on the site corresponding to the URL. 

In short, when a contact clicks on a link in an e-mail, the event is therefore listed twice in the history. 

Do you have the same behaviour?

### How can we reproduce this issue?

Step 1: Sent an e-mail with links
Step 2: The contact opens a link in the received e-mail
Step 3: Check history of this contact

### Relevant log output

```shell
No log
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by dsp76** [comment]
I wasn't able to reproduce it and havn't seen it in any other Mautic instance that we support.
It rather looks like an individual support issue with your instance. Please head over to https://forum.mautic.org/ and open up a discussion there with a good chance people helping you to find the reason.

**Comment by RCheesley** [comment]
Closing the issue due to lack of responses.
